persist/fs/read.go

```diff
@@ -380,21 +388,31 @@ func (r *reader) ReadBloomFilter() (*ManagedConcurrentBloomFilter, error) {
 	)
 }
 
```

Comment:
This method name is a little confusing, it feels like it implies it returns the bytes for an entry when really what it does is just convert arbitrary []byte to (unowned) checked.Bytes. maybe just call it toCheckedBytes or cloneBytes or something

Reply:
I'll call it `entryClonedBytes(...)` ta.